config/default/kustomization.yaml

```diff
@@ -44,3 +45,25 @@ patchesStrategicMerge:
 # Uncomment 'CAINJECTION' in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
 # 'CERTMANAGER' needs to be enabled to use ca injection
 #- webhookcainjection_patch.yaml
+
+patchesJson6902:
```

Comment:
@wfernandes - is there a reason for these removes ? CAPV doesn't add such labels for services or deployments